vscode-udfs/README.md

```diff
@@ -0,0 +1,93 @@
+<!-- title: How to create UDFs with Confluent for VS Code -->
+<!-- description: In this tutorial, learn how to develop, upload, register and execute Flink UDFs with Confluent for VS Code, with step-by-step instructions. -->
+
+# VSCode Flink UDFs
+
+[Install Confluent for VS Code](https://docs.confluent.io/cloud/current/client-apps/vs-code-extension.html).
+
+In the VS Code Activity Bar, click the Confluent icon.
+
+![Activity Bar Selection](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-produce-and-consume/img/activity-bar-selection.png)
+
+If you have many extensions installed, you may need to click … to access 'Additional Views' and select 'Confluent' from the context menu.
+
+![Activity Bar Selection via Additional Views](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-produce-and-consume/img/via-views.png)
+
+In the Side Bar, click 'Connect to Confluent Cloud', and in the permission dialog, click 'Allow'.
+
+![Connect to Confluent Cloud](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-produce-and-consume/img/connect-to-cc.png)
+
+![Allow opening in new window](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-ccloud-quickstart/img/new-window.png)
+
+The web browser opens to the Confluent Cloud login page.
+
+Enter your credentials and click 'Log in'.
+
+![Confluent Cloud Log In Page](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-produce-and-consume/img/login.png)
+
+The web browser shows the 'Authentication Complete' page.
+
+![Head to Confluent Cloud page](https://raw.githubusercontent.com/confluentinc/tutorials/master/vscode-produce-and-consume/img/auth-complete-page.png)
+
+Open the Confluent Cloud Console and [follow these steps](https://docs.confluent.io/cloud/current/get-started/index.html#section-1-create-a-cluster-and-add-a-topic) to create an environment.
+
+Next, create a Flink compute pool by following [these steps](https://docs.confluent.io/cloud/current/flink/operate-and-deploy/create-compute-pool.html).
+
+While that pool is provisioning, return to your editor and confirm that your Confluent Cloud resources are displayed in the Side Bar.
+
+Next, click on "Support" in the sidebar and then "Generate Project From Template". Search "UDFs" and select the Flink UDF template and select your desired directory. 
+
+Navigate to the directory you generate the template in. Time to build the examples!
+
+## Building the examples
```

Comment:
this tutorial doesn't do much guide me to understand "how to develop" a UDF even though it says that above. I think that it should have a section called `Inspect the code` or something that shows users (with code snippets) the important things they need to know:

1. dependencies, i.e. `org.apache.flink:flink-table-api-java` which provides the base classes that you extend to implement a UDF
2. the interface, i.e., "you have to extend ScalarFunction and implement a public method named `eval` etc etc and here is what that looks like for one of the example UDFs that simply sums two integers."